File: backend/app/Models/Slot.php
<?php

namespace App\Models;

use DateTimeInterface;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

/**
 * App\Models\Slots
 *
 * @property int $id
 * @property string $start_time
 * @property string $end_time
 * @property string $status
 * @property int $avail_id
 * @property \Illuminate\Support\Carbon|null $created_at
 * @property \Illuminate\Support\Carbon|null $updated_at
 * @method static \Illuminate\Database\Eloquent\Builder|Slot newModelQuery()
 * @method static \Illuminate\Database\Eloquent\Builder|Slot newQuery()
 * @method static \Illuminate\Database\Eloquent\Builder|Slot query()
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereAvailId($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereCreatedAt($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereEndTime($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereId($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereStartTime($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereStatus($value)
 * @method static \Illuminate\Database\Eloquent\Builder|Slot whereUpdatedAt($value)
 * @mixin \Eloquent
 */
class Slot extends Model
{
    use HasFactory;

}

File: android-app/app/src/main/java/com/example/clinicio/data/remote/slot/Slot.kt
package com.example.clinicio.data.remote.slot

import com.squareup.moshi.Json

data class Slot(
    @field:Json(name = "id") val id: Int = 0,
    @field:Json(name = "start_time") val start_time: String,
    @field:Json(name = "end_time") val end_time: String,
    @field:Json(name = "avail_id") val avail_id: Int,
    @field:Json(name = "created_at") val created_at: String? = null,
    @field:Json(name = "updated_at") val updated_at: String? = null,
    @field:Json(name = "status") val status: Status,
)
